### PR TITLE
Update EIP-1 to fix `discussion-to` and `resolution` URL field

### PR DESCRIPTION
**Pull request by xinbenlv**
Update EIP-1 to fix `discussion-to` and `resolution` URL field

### CODE REVIEW

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -99,7 +99,9 @@ Each EIP must begin with an RFC 822 style header preamble, preceded and followed
 
 ` author:` <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s). Details are below.>
 
-` * discussions-to:` <url>
+` * discussions-to:` \<a url pointing to the destination of official discussion thread for this EIP\>
```
Comment (Arachnid) [inline EIPS/eip-1.md]: "Pointing to the destination" is a bit redundant.

**Comment by xinbenlv** [comment]
Done

**Comment by DanielRX** [comment]
@xinbenlv Would "pointing to `the` official discussion thread" be okay? Just sounds off to me?

**Comment by xinbenlv** [comment]
Sure, @DanielRX

**Pull request merged**